### PR TITLE
windows phone 8 support

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: src/wp8/AccountManager.cs [hunk 1]
```diff
@@ -0,0 +1,171 @@
+﻿using System;
```
Comment (matzew) [inline src/wp8/AccountManager.cs]: mind adding license header to all the (new) files?

**Comment by matzew** [comment]
let's also upate the readme, that after this our lib supports windows phone

**Comment by matzew** [comment]
@sebastienblanc any chance to look here  ?

**Comment by matzew** [comment]
@sebastienblanc @lfryc @edewit any update on this ?

**Comment by sebastienblanc** [comment]
:+1:

**Pull request merged**